test/maps/all_tribes.wmf/scripting/init.lua

```diff
@@ -0,0 +1,182 @@
+-- ================================================
+--       Scenario with all tribes on the map
+-- ================================================
+
+-- This scenario has been prepared for 8 players.
+-- To add a new tribe, copy test_barbarians.lua into a new file and change all the bulding names.
+-- Don't forget to define the tribe in player_names.
+-- Make sure that the building sizes are the same or smaller, and that ports go on port spaces etc.
+
+-- TODO(GunChleoc): Place waterways when we have the Lua interface for them
```

Comment:
We already have it so you can write such a test

Reply:
Placing the worker is broken in connected_road, for busy roads too. I have annotated it in the code and can fix it in this branch.